src/Microsoft.AspNetCore.Mvc.RazorPages/ApplicationModels/PageConventionCollection.cs

```diff
@@ -67,6 +95,34 @@ public IPageApplicationModelConvention AddFolderApplicationModelConvention(strin
             return Add(new FolderApplicationModelConvention(folderPath, action));
         }
 
+        /// <summary>
+        /// Creates and adds an <see cref="IPageApplicationModelConvention"/> that invokes an action on
+        /// <see cref="PageApplicationModel"/> instances for all pages under the specified area folder.
+        /// </summary>
+        /// <param name="areaName">The name of area.</param>
+        /// <param name="folderPath">The path of the folder relative to the Razor Pages root. e.g. <c>/Users/</c></param>
```

Comment:
This is relative to the area root, not the pages root.

Reply:
Yeah, these are bad docs. I'll tidy this up